sources/packages/web/src/views/student/AppStudent.vue

```diff
@@ -34,34 +34,32 @@
           >Applications</v-btn
         >
         <v-btn
-          v-if="hasAuthenticatedStudentAccount"
+          v-show="hasAuthenticatedStudentAccount && showExpandedMenu"
           class="nav-item-label"
           variant="text"
           :to="{ name: StudentRoutesConst.STUDENT_FILE_UPLOADER }"
           prepend-icon="fa:far fa-file-alt"
           >File Uploader</v-btn
         >
         <v-btn
-          v-if="hasAuthenticatedStudentAccount"
+          v-show="hasAuthenticatedStudentAccount && showExpandedMenu"
           class="nav-item-label"
           variant="text"
           :to="{ name: StudentRoutesConst.STUDENT_REQUEST_CHANGE }"
           prepend-icon="fa:far fa-hand-paper"
           >Request a Change</v-btn
         >
-
         <v-menu v-if="isAuthenticated">
           <template v-slot:activator="{ props }">
             <v-btn
-              class="mr-5 nav-item-label"
-              rounded="xl"
-              icon="fa:fa fa-user"
```

Comment:
As discussed, please confirm if there is any change at all to regular large screen view.